tests/unit/test_configuration.py

```diff
@@ -26,6 +26,13 @@ def test_set_sms_region_property_and_check_that_sms_origin_was_updated(sinch_cli
     assert "irritating" in sinch_client_sync.configuration.sms_origin
 
 
+def test_set_sms_region_with_service_plan_id_property_and_check_that_sms_origin_was_updated(sinch_client_sync):
+    sinch_client_sync.configuration.sms_region_with_service_plan_id = (
+        "Herring"
+    )
+    assert sinch_client_sync.configuration._sms_origin_with_service_plan_id.startswith("Herring")
+
+
 def test_set_sms_domain_property_and_check_that_sms_origin_was_updated(sinch_client_sync):
     sinch_client_sync.configuration.sms_region = (
```

Comment:
Oops! I've made a copy/paste typo. It's fixed now.

Reply:
More than the copy/paste typo, I think the test itself needs to be revisited: you did an update in the parameters to test without the need to change the assertion, which is generally not a good sign of the quality of the test. Testing the full value rather than a part of it would be preferable.
It would also make more sense to use meaningful test values instead of a long sentence that makes a hostname invalid.

Reply:
I've made those tests shorter for the ease of reading.
Although those were great Monty Python quotes...

Reply:
This test is already rewritten